edx_sga/sga.py

```diff
@@ -491,6 +495,16 @@ def download_zip(self, student_files):
         # save zip file and return its URL as JSON response
         return Response(json={'zip_url': default_storage.url(default_storage.save(zip_filename, sio))})
 
+    def get_submission_download_status(self, student_id):
+        student_item_dict = self.student_submission_id(student_id)
+        staff_user = User.objects.get(id=self.xmodule_runtime.user_id)
+        return submissions_api.get_download_status(student_item_dict, staff_user)
+
+    def set_submission_as_downloaded(self, student_id):
```

Comment:
Nitpick: I think renaming this to `set_submission_status_to_downloaded` makes more sense